packages/pwa/app/components/drawer-menu/index.jsx

```diff
@@ -260,20 +262,22 @@ const DrawerMenu = ({isOpen, onClose = noop, onLogoClick = noop, root}) => {
                                     </HStack>
                                 </Link>
                             </Box>
-                            <Box>
-                                <LocaleSelector
-                                    {...styles.localeSelector}
-                                    selectedLocale={intl.locale}
-                                    locales={getSupportedLocalesIds()}
-                                    onSelect={(newLocale) => {
-                                        // Update the `locale` in the URL.
-                                        const newUrl = getUrlWithLocale(newLocale, {
-                                            disallowParams: ['refine']
-                                        })
-                                        window.location = newUrl
-                                    }}
-                                />
-                            </Box>
+                            {showLocaleSelector && (
+                                <Box>
+                                    <LocaleSelector
+                                        {...styles.localeSelector}
+                                        selectedLocale={intl.locale}
+                                        locales={supportedLocaleIds()}
```

Comment:
Small error probably while refactoring. We want to use the variable value not call a function.
```suggestion
                                        locales={supportedLocaleIds}
```